virttest/defaults.py

```diff
@@ -1,9 +1,22 @@
+import platform
+ARCH = platform.machine()
+# Set the default machine to i440fx for unidentifed arch also
+# Below line can be removed once support for all possible arch is covered
 DEFAULT_MACHINE_TYPE = "i440fx"
 
+if ARCH in ('ppc64', 'ppc64le'):
+    DEFAULT_MACHINE_TYPE = "pseries"
+elif ARCH in ('x86_64', 'i386'):
+    DEFAULT_MACHINE_TYPE = "i440fx"
```

Comment:
@will-Do Thanks :-)
@xutian : Seeting it to PC might not be common for all, we can have Q35 aswell, This is for enabling the arch dependent defaults, we can decide/change values later aswell.